components/media-feature/Component.jsx

```diff
@@ -81,9 +81,7 @@ function MediaType({ type }) {
 
   return (
     <>
-      <span className="[&>*]:su-h-[20px] [&>*]:su-w-[20px]">
-        {iconMap.get(type)}
-      </span>
+      <span className="*:su-h-20 *:su-w-20">{iconMap.get(type)}</span>
```

Comment:
For direct children selection, there is a out of the box *: selector now
https://tailwindcss.com/blog/tailwindcss-v3-4#style-children-with-the-variant